Topics/Tech_Stacks/JSONRPC.md

```diff
@@ -0,0 +1,25 @@
+### JSON-RPC
+
+## Introduction
+JSON-RPC is a an approach to building APIs for communication between different software systems. Let's compare it to REST, which is the most common way of developing APIs. JSON-RPC is not as popular, but it offers its own advantages. Like REST, JSON-RPC uses JSON for its data format and uses HTTP as a transport protocol. JSON-RPC focuses on remote procedure calls whereas the more popular REST is more focused on resource manipulation. 
```

Comment:
I'm not sure I understand what "remote procedure calls" and "resource manipulation" are in the last sentence. Maybe you could make this comparison to REST a bit simpler, so that someone new to JSON-RPC (and potentially REST) can understand it?